plugins/pastefromword/filter/default.js

```diff
@@ -1085,7 +1085,8 @@
 				symbol = element.attributes[ 'cke-symbol' ];
 
 			element.forEach( function( node ) {
-				if ( !removed && node.value.match( symbol.replace( ')', '\\)' ).replace( '(', '' ) ) ) {
+				// Changing `match` -> `indexOf`, symbol is String not RegExp (#877).
```

Comment:
This comment is true, however if you look at the [`match` docs](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/match):

> A regular expression object. If a non-RegExp object obj is passed, it is implicitly converted to a RegExp by using new RegExp(obj)...

which means it can be used with a String too. I assume what was the intention of this comment, but maybe it will be better to put here something like:

> Since symbol may contains special characters we use simple `indexOf` instead of RegExp which is sufficient (#877).